src/modules/decisions/components/form/FormContainer.tsx

```diff
@@ -299,8 +292,110 @@ class FormContainer extends React.Component<FormContainerProps, FormContainerSta
     });
   }
 
+  handleDecisionMakerLabels = (_data: any) => {
```

Comment:
This should be refactored once the requested index changes are complete (combining the sector_data and searchfield_data fields and making sure they only exist on decisionmaker nodes).